### DISCUSSION TITLE
Error of running the first example twod_axi_coupled

### DISCUSSION THREAD

**Discussion by lipxisi** [arfc/moltres#236] [Q&A Getting started] [answered]
Holle

When I try the first example 2-D axisymmetric core model of MSRE, an error is thrown following and how to solve that?
*** ERROR ***
The following error occurred in the object "fuel", of type "GenericMoltresMaterial".

Unable to open file ../../property_file_dir/newt_msre_fuel_REMXS.txt

Thanks for answer

**Comment by smpark7** [comment, marked as answer]
Hi @lipxisi. Did you run the example from the `moltres/tests/twod_axi_coupled` directory? To be precise, you should set your working directory by `cd moltres/tests/twod_axi_coupled` before running the example.
